clcache.py

```diff
@@ -206,7 +206,7 @@ def computeKey(compilerBinary, commandLine):
         (preprocessedSourceCode, pperr) = preprocessor.communicate()
 
         if preprocessor.returncode != 0:
-            sys.stderr.write(pperr)
+            sys.stderr.write(pperr.decode("utf-8"))
```

Comment:
Could you please open an issue for this decoding issue? Even if you don't have a perfect solution. Maybe we can find out what the problem is together.